packages/api/amplify_api/example/amplify/README.md

```diff
@@ -0,0 +1,8 @@
+# Getting Started with Amplify CLI
```

Comment:
You can delete this file. The provision script creates it, might be able to be added to .gitignore but that's not needed here, just don't want this file in the PR.